packages/algob/src/internal/deployer.ts

```diff
@@ -68,8 +69,8 @@ class DeployerBasicMode {
     return this.cpData.precedingCP[this.networkName]?.asa ?? new Map();
   }
 
-  get asc (): Map<string, ASCInfo> {
-    return this.cpData.precedingCP[this.networkName]?.asc ?? new Map();
+  get ssc (): Map<string, SSCInfo> {
```

Comment:
kind question: Algorand smart contracts are of two types `stateless` and `stateful`. Is it okay to change from ASC (which is a generic name) to SSC? What if a user is executing a run script with a stateless teal, he would still use `get ssc()` to retrieve cp data, right? What do you think?

Reply:
No, i don't think so because as per SPEC, `stateless` contracts are loaded and used as DelegatedLsig , loadLsig and fundLsig. 
But we can change it , what do you suggest ?

Reply:
ah, i see. If it's independent from `stateless` then we can use ssc. Feel free to resolve.